restart_process/Dockerfile

```diff
@@ -7,4 +7,8 @@ RUN apk add build-base
 RUN git clone https://github.com/eradman/entr.git /entr
 RUN cd /entr; ./configure; CFLAGS="-static" make install
 
+FROM alpine:3.12
```

Comment:
if you wanna get crazy, you could try making this `scratch`

Reply:
Good idea! Done